integr8ly/integreatly-operator
# MGDAPI-5152 - Changes to support OpenShift v4.13

# Issue link
[MGDAPI-5152](https://issues.redhat.com/browse/MGDAPI-5152)

# What
- Update various dependencies to support installation on OpenShift v4.13 - primarily to explicitly set the GrpcPodConfig in CatalogSource CR's.
- This relies on an update to do the same in the Observability Operator, as well as a dependency update from 3scale Operator.
	- Observability Operator PR https://github.com/redhat-developer/observability-operator/pull/197
	- 3scale Operator PR https://github.com/3scale/3scale-operator/pull/805


# Outstanding
* Once the above PR's are merged I will need to update the go.mod replaces to point to the official versions of the above
* This is currently using a custom index for the observability operator with the required changes. Once the changes have been merged the new release of the operator will need to be consumed in the installation.yaml

# Verification steps

* Provision a v4.13 cluster and deploy RHOAM from this branch

```
LOCAL=false make cluster/prepare/local
LOCAL=false USE_CLUSTER_STORAGE=true make code/run
```

* Watch the RHMI CR during installation - it should complete successfully
* Verify no alerts are firing
* Trigger uninstall and watch RHMI CR - expect RHOAM to successfully uninstall
```
oc delete rhmi rhoam -n redhat-rhoam-operator
```
